L1Trigger/L1TTrackMatch/plugins/L1TrackVertexAssociationProducer.cc

```diff
@@ -353,6 +361,9 @@ void L1TrackVertexAssociationProducer::produce(edm::StreamID, edm::Event& iEvent
   TTTrackDeltaZMaxSelector deltaZSel(deltaZMaxEtaBounds_, deltaZMax_);
   TTTrackWordDeltaZMaxSelector deltaZSelEmu(deltaZMaxEtaBounds_, deltaZMax_);
 
+  TTTrackLinkLimitSelector linkLimitSel{};  // Brace for the Most Vexing Parse
+  TTTrackLinkLimitSelector linkLimitSelEmu{};
```

Comment:
why not simply
```suggestion
  TTTrackLinkLimitSelector linkLimitSel;
  TTTrackLinkLimitSelector linkLimitSelEmu;
```
?

Also, why two of them?
`TTTrackLinkLimitSelector` does not have any data members, so the two selectors are effectively identical.